server/routes/team.router.js

```diff
@@ -143,6 +147,28 @@ router.put('/bye', rejectUnauthenticated, (req, res) => {
     })
 })
 
+router.put('/paid', rejectUnauthenticated, (req, res) => {
+  console.log('req.body', req.body);
+  
+  let queryText = `
```

Comment:
@patrickrnelson this endpoint appears to be missing Authorization controls. Only admins should be able to mark leagues as paid